src/index.html

```diff
@@ -7,6 +7,51 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="week week--indentions">
+      <div class="week__day">Monday</div>
+      <div class="week__day">Tuesday</div>
+      <div class="week__day">Wednesday</div>
+      <div class="week__day">Thursday</div>
+      <div class="week__day">Friday</div>
+      <div class="week__day">Saturday</div>
+      <div class="week__day">Sunday</div>
+    </div>
+
+    <div class="calendar
+      calendar--indentions
+      calendar--month-length--31
+    ">
+      <div class="calendar__day calendar__day--start-day--sun"></div>
```

Comment:
check task description: start-day should be a modifier for calendar (not for calendar__day)